packages/grafana-toolkit/src/cli/tasks/plugin.utils.ts

```diff
@@ -131,28 +138,28 @@ const prepareRelease = useSpinner<any>('Preparing release', async ({ dryrun, ver
 
 interface GithubPluglishReleaseOptions {
   commitHash?: string;
-  recreate?: boolean;
+  dev?: boolean;
```

Comment:
what is in dev here?  is this referring to your testing?  is it really dryRun or something?

Reply:
No, if selected, you create a dev branch with a build, rather than a release branch (ie: release-1.4.0-dev", and release a dev release called 1.4.0-dev. Not really specific yet, but more options available as a result of using the rest api.